### ISSUE TITLE
Issue with painkiller

### ISSUE THREAD

**Issue by Ex0r** [mythicrp/mythic_hospital#5]
I am using this with ESX (I havent integrated it yet, but am testing it out). The medkit, gauze and bandages work and show animations, but whenever I use any of the painkillers (vicodin, morphine, hydrocodone) the animation doesn't play and nothing happens. Am I doing something wrong here?

**Comment by Alzar** [comment]
Are you using the ESX fork of this? Or have you modified it yourself?

The stand-alone version doesn't implement any items so not sure, best suggestion would be opening an issue on the ESX fork and seeing of those who're more familiar with ESX could help you as I haven't ever used it so no idea.

**Comment by Ex0r** [comment]
No, I'm not. Was just using the standalone resource to see how it worked. It's only the painkillers itself that don't work, the rest does so figured maybe it was a bug. 

I didn't even know there was an ESX fork of this resource tbh.

**Comment by Alzar** [comment]
Are you getting an error at all?

And yep, few other community members have ported it to ESX and have been maintaining it. Can find it [here](https://github.com/Nowimps8/mythic_hospital)

**Comment by Ex0r** [comment]
No errors at all. The progress bar doesn't show, the animation doesnt show, and the effects don't show. I also noticed that the item doesn't actually get removed from my inventory either, even if it's used.

It's weird I have no idea what's up with it. I also thought I understood the uses but I guess I'm wrong. 

Hydrocodone, vicodin, morphine = treat pain,
Medkit, first aid, bandage = treat injuries
Gauze = stops bleeding 

Are those correct? It's possible maybe the pills are working, but not animating or showing the progress bar, but I took like six vicodin when it said I was pained and it didn't do anything. Hard to tell since it doesn't subtract from my inventory which I'll fix tomorrow.
